src/Dashboard/Dashboard.jsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Dashboard.css";
+import jwt_decode from "jwt-decode";
+
+const Dashboard = () => {
+  const getToken = JSON.parse(localStorage.getItem("data"));
+  const decodeToken = jwt_decode(getToken);
+  const { firstName, email } = decodeToken.user;
+  console.log(decodeToken);
```

Comment:
Remove this console.log if it's no longer needed

Reply:
Removed already boss